ros/img2vec/scripts/img2vec.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+import sys
+import rospy
+from sensor_msgs.msg import Image
+from rospy.numpy_msg import numpy_msg
+from cv_bridge import CvBridge, CvBridgeError
+import cv2
+ 
+import torch
+import torch.nn as nn
+import torchvision.models as models
+import torchvision.transforms as transforms
+from torch.autograd import Variable
+from PIL import Image
+ 
+class image_converter:
```

Comment:
CamelCase is more Pythonic.

```suggestion
class ImageConverter:
```